app/models/lookup_value.rb

```diff
@@ -24,6 +24,10 @@ class LookupValue < ActiveRecord::Base
   scoped_search :on => :match, :complete_value => true
   scoped_search :in => :lookup_key, :on => :key, :rename => :lookup_key, :complete_value => true
 
+  def value_present?
+    !!value.nil?
```

Comment:
yeah, forgot how validators work. that's why tests failed, pushing fix in a
few minutes, once tests pass on my machine